example/lib/main.dart

```diff
@@ -39,6 +40,7 @@ class HomePage extends StatelessWidget {
               Tab(child: Text("Numbers")),
               Tab(child: Text("Dropdown")),
               Tab(child: Text("Custom Builder")),
+              Tab(child: Text("Scroll")),
```

Comment:
please name it "Scrollable Numbers" and put it in the order directly after "Numbers"